NOTICES.md

```diff
@@ -0,0 +1,1133 @@
+    conditions and limitations in section 3, each contributor grants you a
+    non-exclusive, worldwide, royalty-free license under its licensed patents to
+    make, have made, use, sell, offer for sale, import, and/or otherwise dispose of
+    its contribution in the software or derivative works of the contribution in the
+    software.
+
+    3. Conditions and Limitations
+    
+    (A) No Trademark License- This license does not grant you rights to use any
+    contributors' name, logo, or trademarks.
+    
+    (B) If you bring a patent claim against any contributor over patents that you
+    claim are infringed by the software, your patent license from such contributor
+    to the software ends automatically.
+    
+    (C) If you distribute any portion of the software, you must retain all
+    copyright, patent, trademark, and attribution notices that are present in the
+    software.
+    
+    (D) If you distribute any portion of the software in source code form, you may
+    do so only under this license by including a complete copy of this license with
+    your distribution. If you distribute any portion of the software in compiled or
+    object code form, you may only do so under a license that complies with this
+    license.
+    
+    (E) The software is licensed "as-is." You bear the risk of using it. The
+    contributors give no express warranties, guarantees or conditions. You may have
+    additional consumer rights under your local laws which this license cannot
+    change. To the extent permitted under your local laws, the contributors exclude
+    the implied warranties of merchantability, fitness for a particular purpose and
+    non-infringement.
+      
+## Coypu & Coypu.NUnit
+https://github.com/featurist/coypu  
+
+    MIT License
+    
+    Copyright (c) 2018 Adrian Longley & Contributors
+    
+    Permission is hereby granted, free of charge, to any person obtaining a copy
+    of this software and associated documentation files (the "Software"), to deal
+    in the Software without restriction, including without limitation the rights
+    to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+    copies of the Software, and to permit persons to whom the Software is
+    furnished to do so, subject to the following conditions:
+    
+    The above copyright notice and this permission notice shall be included in all
+    copies or substantial portions of the Software.
+    
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+
+Dapper
```

Comment:
All of the items from here down need to be reformatted by adding the `##` markdown to project name to create a header, and indenting the license text by four spaces to cause it to be a quoted blocktext. For example:
```suggestion
## Dapper
```